index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    Hello World, we are software developers!
```

Comment:
Please add text in an h1 tag. 😊

Reply:
- [ ] Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
For example : "Kindly, I recommend you to use `h1` tag  for first heading element , that would improve the HTML structure and make the page more accessible"